upstream.go

```diff
@@ -888,6 +904,8 @@ func (uc *upstreamConn) handleMessage(msg *irc.Message) error {
 				Params:  []string{dc.marshalNick(uc, nick), dc.marshalChannel(uc, channel)},
 			})
 		})
+	case "TAGMSG":
+		// Ignore
```

Comment:
The spec says this should be handled just like a `PRIVMSG` (ie. relayed). We should add a `// TODO: relay to downstream accepting message-tags`.

Reply:
Here, you can apply this suggestion ;)

```suggestion
		// TODO: relay to downstream accepting message-tags
```

Reply:
Please don't use GitHub suggestions. Those add fixup commits instead of amending the original commit.

Reply:
Oh ok, didn't know.  I see how this is not wanted here.